build-coordinator/src/main/java/org/jboss/pnc/core/builder/BuildSetTask.java

```diff
@@ -126,7 +126,22 @@ public void addBuildTask(BuildTask buildTask) {
         buildTasks.add(buildTask);
     }
 
-    public Integer getId() {
+    /**
+     * Get the build task which contains the given build configuration
+     * 
+     * @param buildConfig
+     * @return The build task with the matching configuration, or null if there is none
+     */
+    public BuildTask getBuildTask(BuildConfiguration buildConfig) {
```

Comment:
I think it's ok here.  The BuildSetTask contains a bunch of BuildTasks, so you can have convenient ways to access them.  In a utility class you would still have to pass both the BuildSetTask and the BuildConfiguration, and then do the same operation.